fee-monitor/src/AccountState.ts

```diff
@@ -0,0 +1,81 @@
+import { PlatformAddress } from "codechain-sdk/lib/core/classes";
+import { sdk } from "./config";
+import { decodePlatformAddress, decodePlatformAddressfromPubkey } from "./Stake";
+
+const RLP = require("rlp");
+
+export enum AccountState {
+    Eligible,
+    Candidate,
+    Validator,
+    Banned,
+    Jailed,
+}
+
+type QueyrableStates = Exclude<AccountState, AccountState.Eligible>;
```

Comment:
It seems a typo `QueyrableStates`.
`QueyrableStates` -> `QueryableStates`

Reply:
Thank you. I fixed it.